Gemfile

```diff
@@ -1,3 +1,19 @@
 source 'https://rubygems.org'
 
 gemspec
+
+group :development do
+  gem 'pry'
```

Comment:
is pry something we want to leave in here?

Reply:
is it not just for debugging purposes?

Reply:
yeah correct - but it's useful to have the gem defined in the development group.

Otherwise we have to add it each time we want to run a pry.

Also, fwiw - Gemfile != Gemspec. So gems defined here will not end up as dependencies of the gem.